services/common/src/components/projectSummary/ProjectManagement.tsx

```diff
@@ -0,0 +1,195 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  clearProjectSummaryMinistryComments,
+  createProjectSummaryMinistryComment,
+  fetchProjectSummaryMinistryComments,
+} from "@mds/common/redux/actionCreators/projectActionCreator";
+import { getProjectSummaryMinistryComments } from "@mds/common/redux/selectors/projectSelectors";
+import { faLock } from "@fortawesome/pro-regular-svg-icons";
+import { useFeatureFlag } from "@mds/common/providers/featureFlags/useFeatureFlag";
+import MinistryCommentPanel from "@mds/common/components/comments/MinistryCommentPanel";
+
+const { Paragraph, Title } = Typography;
+
+const unassignedProjectLeadEntry = {
+  label: "Unassigned",
+  value: null,
+};
+
+export const ProjectManagement: FC = () => {
+  const dispatch = useDispatch();
+  const systemFlag = useSelector(getSystemFlag);
+  const isCore = systemFlag === SystemFlagEnum.core;
+
+  const { isFeatureEnabled } = useFeatureFlag();
+
+  const { status_code, project_summary_guid, project_lead_party_guid } = useSelector((state) =>
+    getFormValues(FORM.ADD_EDIT_PROJECT_SUMMARY)(state)
+  );
+
+  const ministryComments = useSelector(getProjectSummaryMinistryComments);
+
+  useEffect(() => {
+    if (project_summary_guid) {
+      dispatch(fetchProjectSummaryMinistryComments(project_summary_guid));
+    }
+
+    return () => {
+      dispatch(clearProjectSummaryMinistryComments());
+    };
+  }, [project_summary_guid]);
+
+  if (!isCore) {
+    return null;
+  }
+
+  const isNewProject = !project_summary_guid;
+  const isProjectLeadAssigned = Boolean(project_lead_party_guid);
+  const projectSummaryStatusCodes = useSelector(getDropdownProjectSummaryStatusCodes);
+
+  const projectLeads: IGroupedDropdownList = useSelector(getDropdownProjectLeads);
+  const projectLeadData = [unassignedProjectLeadEntry, ...projectLeads[0]?.opt];
+
+  const submitComment = async ({ comment }) => {
+    await dispatch(createProjectSummaryMinistryComment(project_summary_guid, { content: comment }));
+  };
+
+  return (
+    <>
+      {isFeatureEnabled(Feature.MAJOR_PROJECT_REFACTOR) ? (
+        <div>
+          <Typography.Title level={3}>Project Management</Typography.Title>
+          <Paragraph>
+            Need help with this feature? Here&apos;s the <a>guide</a> to help you learn more or get
+            step-by-step instructions.
+          </Paragraph>
+          <Title level={3} className="color-primary">
+            EMLI Status
+          </Title>
+          <Title level={4}>Project Status</Title>
+          <Paragraph>
+            Project status is shared with the proponent via MineSpace. Changing the status will
+            notify the project’s Primary Contact.
+          </Paragraph>
+          {status_code && (
+            <Field
+              id="status_code"
+              name="status_code"
+              required
+              component={RenderSelect}
+              data={projectSummaryStatusCodes}
```

Comment:
Something to watch out for- the `required` attribute will put on the indicator, but it won't do the validating. Probably need a required validator in here too.